src/twisted/internet/iocpreactor/abstract.py

```diff
@@ -344,7 +346,7 @@ def loseConnection(self, _connDone=failure.Failure(main.CONNECTION_DONE)):
             if self._writeDisconnected:
                 # doWrite won't trigger the connection close anymore
                 self.stopReading()
-                self.stopWriting
+                self.stopWriting()
```

Comment:
🤦

Reply:
Hmm. Codecov is still complaining about this. Possibly the codecov-push thing *needs* to be landed on trunk first?